bin/reth/src/commands/debug_cmd/build_block.rs

```diff
@@ -247,6 +247,7 @@ impl<C: ChainSpecParser<ChainSpec = ChainSpec>> Command<C> {
 
         let payload_builder = reth_ethereum_payload_builder::EthereumPayloadBuilder::new(
             EthEvmConfig::new(provider_factory.chain_spec()),
+            EthereumBuilderConfig::new(Default::default()),
```

Comment:
perhaps it would be better if EthereumBuilderConfig had `Default`?

Reply:
ideally, this would be `default_extradata()`, but it's not available in the crate where `EthereumBuilderConfig ` is defined, so i'd rather not have the default

Reply:
I see